packages/money_transfer_service/routes/get.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+const MoneyTransfer = mongoose.model('tMoneyTransfer');
```

Comment:
You have to model the table, so you should write table name here:
```suggestion
const MoneyTransfer = mongoose.model('tmoneytrans');
```